### DISCUSSION TITLE
how to show ProgressBar for each image during uploading when using fileupload

### DISCUSSION THREAD

**Discussion by allaghi** [primefaces/community#1037] [PrimeVue] [answered]
Dears,
Have a nice day,
I am wondering if anyone has idea how to show ProgressBar for each image during uploading files and images,
currently it just shows pending and completed.

Thanks in advance.

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

I want to reference this [example](https://primevue.org/fileupload/#template). `totalSizePercent` can be changed when `uploadEvent (uploadCallback)` event is called.
